introduction.md

```diff
@@ -3,3 +3,9 @@ HELLO
 ---
 Hi, I'm jahrik.  Junior samurai.
 My favorite path so far from Code School is the Ruby and git in the Electives.
+
+Deadly skills:
+* git
+* ruby
+* python
+* confluence
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?